perun-base/src/main/java/cz/metacentrum/perun/core/api/exceptions/InvalidConsentStatus.java

```diff
@@ -0,0 +1,36 @@
+package cz.metacentrum.perun.core.api.exceptions;
+
+/**
+ * This exception is thrown when trying to set consent status on an invalid value.
+ *
+ * @author Sarka Palkovicova
+ */
+public class InvalidConsentStatus extends PerunException {
```

Comment:
```suggestion
public class InvalidConsentStatusException extends PerunException {
```
to match convention of other custom created exceptions.